server/auth/validator.js

```diff
@@ -0,0 +1,32 @@
+const { INTERPRETER, USER } = require("./roles");
+function validateEmailAddress(email) {
+  var expression = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  if (expression.test(String(email).toLowerCase())) {
```

Comment:
I don't think you need `String` here, you can just use `email.toLowerCase()`, email is already a string I think?

Reply:
It is changed